### PR TITLE
Finish calendar

### PR DESCRIPTION
**Pull request by AntonBaranskyi**
[DEMO LINK](https://AntonBaranskyi.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month--length-28 calendar--start--day--sat">
```
Comment (And678) [inline src/index.html]: [Checklist #2](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) - [BEM] - Make sure to follow BEM naming convention for complex modifiers: `block-name--modifier-name--modifier-value`;
```suggestion
      <div class="calendar calendar--month-length--28 calendar--start-day--sat">
```

Comment (And678) [reply src/index.html]: In this case
modifier-name = month-length
modifier-value = 28

Comment (And678) [reply src/index.html]: Don't forget to update scss too

**Review by And678 — CHANGES_REQUESTED** [review]
Make sure it works in all cases of start-day and month-length, even for invalid ones ([checklist 10](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md)).

On picture: start-day = mon, length = 31:

image

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
image
please, make sure that after modificators changing all works as expected

**Pull request closed without merging**